src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
It is common practice to apply `box-sizing: border-box` for all elements via `*` selector